src/main/java/step2/Calculator.java

```diff
@@ -0,0 +1,48 @@
+package step2;
+
+public class Calculator {
+    public int sum(int num1, int num2) {
+
+        return num1+num2;
+    }
+
+    public int subtract(int num1, int num2) {
+        return num1-num2;
+    }
+
+    public int multiply(int num1, int num2) {
+        return num1*num2;
+    }
+
+    public int divide(int num1, int num2) {
+        return num1/num2;
+    }
+
+    public int calculate(String input) {
+        //빈문자열일 경우
+        if(input.trim().length() == 0 || input == null){
```

Comment:
```suggestion
        if (input.trim().length() == 0 || input == null){
```
그리고 `if` 조건절은 앞에서부터 순차적으로 비교하기 때문에 정말 `null` 값이 들어오는 경우
`input.trim()`에서 `NPE`가 발생합니다 😄 
`null` 먼저 조건이 비교 될 수 있도록 수정이 필요할 것 같습니다 😄